.eslintrc.js

```diff
@@ -0,0 +1,94 @@
+module.exports = {
+    root: true,
+    parser: '@typescript-eslint/parser',
+    parserOptions: {
+        project: './tsconfig.json',
+        // sourceType: "module"
+    },
+    plugins: ['@typescript-eslint', 'react', 'import', 'functional', 'sonarjs'],
+    extends: [
+        'eslint:recommended',
+        'plugin:@typescript-eslint/eslint-recommended',
+        'plugin:@typescript-eslint/recommended',
+        'plugin:react/recommended',
+        'plugin:sonarjs/recommended',
+        'prettier',
+        'prettier/@typescript-eslint',
+        'plugin:prettier/recommended',
+    ],
+    rules: {
+        'no-case-declarations': 'off',
+        'no-inner-declarations': 'off',
```

Comment:
can we share this setup into a common repo ie. https://github.com/pagopa/eslint-config-pagopa/tree/add-typescript-recommended-config ?

Reply:
@gunzip should we just import that configuration? the above config has been replicated from `io-app`, what's our reference config?

Reply:
there isn't a reference config. that's why I'm suggesting to take this as a reference and put the config into a shared repo (so we can reuse the work here in other projects without copy & paste)

Reply:
- I agree to migrate to azure pipelines.. I thought I needed to preserve CI/CD support already in place in io-pay repo
- I'll open a PR on the repo you linked, can it work as a sharing strategy?

Reply:
@gunzip that's out of scope, I suggest to make that task as part of the migration to eslint